README.md

```diff
@@ -6,6 +6,13 @@
 
 An MCP server that provides a tool for sending transactional emails via Mailtrap
 
+## Quick Install
```

Comment:
I would move `VS Code Insiders` here as well and we can delete `#### One-click installation (VS Code / VS Code Insiders)` section